### DISCUSSION TITLE
input.conf not working in ver 24.5.0.10337 portable x64 in windows 10

### DISCUSSION THREAD

**Discussion by bhavtosh** [smplayer-dev/smplayer#1088] [Q&A]
Hello all,

I am using SMPlayer_24.5.0.10337 portable x64. in win 10

I just uncommented the F1, F2 and F3 with below commands in input.conf located at D:\Tools\SMPlayer_24.5.0.10337\mplayer\mplayer but smplayer does not seem to locate or apply changes made in input.conf.

F1 add_filter lavfi=[compand=attacks=0:points=-80/-80|-10/-5|0/0]
F2 del_filter lavfi
F3 show_osd

kindly help me to fix this issue. Good day.

**Comment by smplayer-dev** [comment]
`input.conf` is not used by smplayer.
You need to enable the option "Run mplayer in its own window" in preferences -> advanced -> mplayer, and maybe then it would work.

**Reply by bhavtosh** [reply]
thanks. I tried doing it and it detached both programs completely. 

Now I have two windows.

One is smplayer being the unsed one and the mpv the active one with no toolbar or way to be configured via right click flexibility or input keystrokes.

WELL smplayer does provide the option of applying filters via the same path "preferences -> advanced -> mplayer/mpv" BUT the issue is that it applies to all video files that i play.

I want to apply different filters on videos of my choice of video files and hence need a functionality like input.conf

So kindly advise further to achieve this in smplayer. good day

**Comment by smplayer-dev** [comment]
You can use View -> Information & properties -> Options for mplayer/mpv to pass options for the file that is currently playing, this won't affect the rest of videos.
